api2-war/src/main/java/eu/europeana/api2/v2/web/controller/ObjectController.java

```diff
@@ -380,25 +419,31 @@ private ModelAndView generateSchemaOrg(FullBean bean, RequestData data) {
     }
 
     private ModelAndView generateJsonLd(FullBean bean, RequestData data, HttpServletResponse response) {
-        String jsonld;
+        OutputStream outputStream = new ByteArrayOutputStream();
```

Comment:
Outputstream is never closed. This will lead to problems!

Reply:
Made it auto closeable

Reply:
Ok, that will work. Checking if something is null and then closing in a finally block works fine and is great for non-autocloseables. The great thing about autoclosables wuch as Outputstream is that everything is done for you once you put it in a try() block, so I can recomend to always use this if possible.